python/tvm/autotvm/task/relay_integration.py

```diff
@@ -25,14 +25,30 @@
 import logging
 
 
-from ... import target as _target
-
 from .task import create
 from .topi_integration import TaskExtractEnv
 
 logger = logging.getLogger('autotvm')
 
 
+def _build(func,
+           target,
+           target_host,
+           params):
+    """ Helper to build VTA properly.
+    """
+
+    from tvm import relay
+
+    if "vta" in str(target):
```

Comment:
Let's merge the PR, but can you put this code in a more appropriate place?

Reply:
Good point, I'll add a TODO so we can address it in a future issue.

Reply:
I am a bit confused about why the change is necessary. my feeling is that the original code is better.

Reply:
@tqchen are you referring to the `_build()` function? since we're building inside of a thread (see below), we'll need to enter the vta build config environment.

Reply:
I see what you mean; I reverted back to `target.device_name == "vta"` and fixed the error that I was getting in the CI: the issue was that some tests in the `test_graph_tuner_core.py` were not initializing the tvm target correctly. This is now fixed.